tools/extra/fpgadiag/CMakeLists.txt

```diff
@@ -107,9 +107,18 @@ set_property(DIRECTORY PROPERTY
     ADDITIONAL_MAKE_CLEAN_FILES
         "${CMAKE_CURRENT_SOURCE_DIR}/build"
         "${CMAKE_CURRENT_SOURCE_DIR}/dist"
-        "${CMAKE_CURRENT_SOURCE_DIR}/test_vfio.o"
 )
 
+if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS 5.0)
+   set_source_files_properties(src/eth_group.cpp  PROPERTIES COMPILE_FLAGS -fno-lto)
+endif()
+set_target_properties(opaeuio PROPERTIES INSTALL_RPATH_USE_LINK_PATH TRUE)
```

Comment:
Hmm, if this is something that we need to set, it would be best if it appeared in the libopaeuio CMakeLists.txt.  What is this doing?  How will it affect other tools that need this library?

Reply:
removed Compile flag  set_source_files_properties(src/eth_group.cpp  PROPERTIES COMPILE_FLAGS -fno-lto)

Reply:
Sorry, I was referring to line 112:
```bash
set_target_properties(opaeuio PROPERTIES INSTALL_RPATH_USE_LINK_PATH TRUE)
```